lxd/devices.go

```diff
@@ -689,7 +689,7 @@ func deviceEventListener(stateFunc func() *state.State) {
 
 			s := stateFunc()
 
-			if !s.OS.CGInfo.Supports(cgroup.NetPrio, nil) {
+			if e[1] != "add" {
```

Comment:
Could you add a comment here explaining why we ignore events other than add? Thanks

Reply:
As you can see this change in a separate commit

Reply:
Yeah I agree, I thought a comment in the code would be useful for future readers.